ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/metrics/DeltaFilesMetricReporter.java

```diff
@@ -45,32 +45,46 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
-import java.util.Comparator;
+import java.util.*;
```

Comment:
Nit: I think it's best to import classes one-by-one

Reply:
fixed